app/src/main/res/layout/item_swipeable_card.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This is the layout file we'll modify to add more information to the swipe card like breed, etc.